README.md

```diff
@@ -1,30 +1,51 @@
-<h1 align="center">A official AsyncAPI React Component</h1>
+# AsyncAPI React Component
 
 ## Overview
 
-A official [React](https://reactjs.org/) component for AsyncAPI specification. It allows to validation and render AsyncAPI specification and use own styles.
+This project is an official [React](https://reactjs.org/) component for AsyncAPI specification. It allows you to validate and render AsyncAPI specification using your own styles.
 
-## Philosophy
+### Project structure
 
+This repository has the following structure:
 
+```
+  ├── docs                        # Directory with project-related documents
+  ├── library                     # Structure of the AsyncApi React component
+  │    ├── src                    # Source code of the AsyncApi React component
+  │    │    ├── components        # Source code of generic components used in the "containers" directory
+  │    │    ├── config            # Configuration of the AsyncApi React component
+  │    │    ├── containers        # Subcomponents for specific parts of the AsyncApi React component
+  │    │    ├── helpers           # Various helper functions
+  │    │    └── theme             # Theme-related files for the AsyncApi React component
+  │    └── test                   # Tests for the AsyncApi React component
+  └── playground                  # Structure of the Playground application for the AsyncApi React component
+       ├── public                 # Fonts, images, and icons used in the Playground application
+       ├── src                    # Source code of the Playground application
+       │    ├── common            # Various helper functions including mocks
+       │    └── components        # Components used in the Playground application
+       └── test                   # Tests for the Playground application
+```
+
+## Prerequisites
+
+Use the following tools to develop the AsyncApi React component:
 
-## Live Demo
+* React (version 16.0.0 or higher)
+* [Styled-components](https://github.com/styled-components/styled-components) (version 4.0.0 or higher)
+* TypeScript (version 3.0.0 or higher)
 
```

Comment:
Please add here a new section:


## Playground

This repository comes in with a Playground that you can use to see the component in action and play with it before you use it in your application.

### Installation

Run this command to install all dependencies:
```
npm install
npm run bootstrap
cd library
npm run build:lib
cd ..
```

### Usage

Run this command to start the local server with the Playground application:

`npm run playground`

Reply:
I think this idea is bad, because Playground is only for development and instructions how prepare dev env is already in DEVELOPMENT.md. Lerna by `npm start` in core package.json run in parallel start scripts for playground and library and by your proposition after each change in library dir, the developer would have to run `npm run build:lib` so this will be very frustrating.

Reply:
we need this section here as playground is not only for dev but for now us also a only usage example that readme reader can quickly use